smart-client-jersey/src/main/java/com/emc/rest/smart/jersey/SmartFilter.java

```diff
@@ -74,10 +75,13 @@ public ClientResponse handle(ClientRequest request) throws ClientHandlerExceptio
 
             return response;
         } catch (RuntimeException e) {
-
             // capture requests stats (error)
-            host.callComplete(true);
-            host.connectionClosed();
+            if (e instanceof SmartClientException && ((SmartClientException) e).getErrorType().equals(SmartClientException.ErrorType.Client)) {
+                host.callComplete(false);
```

Comment:
we have to call `connectionClosed()` in both cases, because the underlying client has already closed the connection due to an error condition (> 399)... so maybe we should define a boolean here to capture whether this was a server error, and send that boolean to `callComplete()` - then both host calls can be outside of any conditional